lib/tasks/factory_girl.rake

```diff
@@ -1,15 +0,0 @@
-namespace :factory_girl do
-  desc "Verify that all FactoryGirl factories are valid"
-  task lint: :environment do
-    if Rails.env.test?
-      begin
-        DatabaseCleaner.start
-        FactoryGirl.lint
-      ensure
-        DatabaseCleaner.clean
-      end
-    else
-      system("bundle exec rake factory_girl:lint RAILS_ENV='test'")
-    end
-  end
-end
```

Comment:
It was erring on `bin/rake` so I removed it. If we do start using it. (Which I doubt) We can add it back in at that time.